package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "python-tools",
   "main": "./lib/python-tools",
-  "version": "0.6.8",
+  "version": "0.7.0",
```

Comment:
could you revert this change please? I will find increase the version when I find the right number of changes to include with the next release :)